### ISSUE TITLE
[Feature Request]: QOL

### ISSUE THREAD

**Issue by DanMeadWasTaken** [Cheos137/ArmorpointsPlusplus#63]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest version currently available?

- [X] I am using the latest version currently available

### Description

1. An overlay on top of the armor bar to show thorns enchant
2. flashing white (with configurable animation speed) when mending an item
3. flashing red when durability of a piece is low
 -these last two could either be for the specific piece, or for the whole bar- the whole bar would likely be easier to do since the bars stack

### Solution

the thorns icon could be like a prickly look to the bar which changes color (like the other bars) if it goes over the existing limit of the bar.

and the colors for mending could change from say red (low durability) to green (about to be full)

maybe a similar thing with the low durability- except just flash a slow orange when mid-way to low then red when really low

### Alternatives

maybe just a triangle icon on the top left corner of the armor icon for thorns?

### Additional Context

Basically, like this [mod](https://modrinth.com/mod/detail-armor-bar)

**Comment by Cheos137** [comment]
sounds cool!
will probably target this at v3.2.0
